spec/features/user_browses_loan_requests_spec.rb

```diff
@@ -1,22 +1,32 @@
 require "rails_helper"
 
-RSpec.feature "user browses items" do
-  let(:create_item) { LoanRequest.create( ) }
-
-  context "unauthenticated user" do
-    xscenario "can browse the items" do
-      # create(:item)
-      visit "/menu"
-      expect(current_path).to eq(menu_path)
-      expect(page).to have_content("Sushi")
+RSpec.feature "unauthenticated user browses loan requests" do
+  let(:create_item) { LoanRequest.create(title: "Farm Tools",
+                                      description: "help out with the farm tools",
```

Comment:
Line is too long. [82/80]<br>Align the elements of a hash literal if they span more than one line.